src/Apps/Artwork/ArtworkApp.tsx

```diff
@@ -46,23 +45,11 @@ export class ArtworkApp extends React.Component<Props> {
   componentDidMount() {
     this.trackPageview()
     this.trackProductView()
-    this.trackABTest() // TODO: Remove after AB test
   }
 
-  trackABTest() {
-    const { tracking } = this.props
-    const { CLIENT_NAVIGATION } = sd
-
-    const experiment = "client_navigation"
-    const variation = CLIENT_NAVIGATION
-    tracking.trackEvent({
-      action_type: Schema.ActionType.ExperimentViewed,
-      experiment_id: experiment,
-      experiment_name: experiment,
-      variation_id: variation,
-      variation_name: variation,
-      nonInteraction: 1,
-    })
+  componentDidUpdate() {
+    this.trackPageview()
+    this.trackProductView()
```

Comment:
Is it _always_ correct to treat each "update" as a page view, or are some page updates more minor or internal to a single page?

Reply:
In this case, I believe it's valid to do anything we are doing in `componentDidMount`, in `componentDidUpdate` as well.

Any page update, like clicking on and being navigated to a new artwork from an existing artwork in client-side routing, will trigger this (and not the `componentDidMount` method), and the intent there is to execute that code in each of those occurrences.

When other interactions on the page happen, like someone interacting with a filter on the artist page, that won't trigger this method in the app component at all. So I think this is correct.